test-data/ReactA11yProptypes/FailingTestInputs/notAllowUndefined.tsx

```diff
@@ -0,0 +1,4 @@
+import React = require('react');
```

Comment:
Seems Typescript error, then skip this test.

 

From: bounces+848413-bc81-supersingerman=126.com@sgmail.github.com [mailto:bounces+848413-bc81-supersingerman=126.com@sgmail.github.com] On Behalf Of t-ligu
Sent: Tuesday, September 13, 2016 4:49 PM
To: Microsoft/tslint-microsoft-contrib <tslint-microsoft-contrib@noreply.github.com>
Cc: Liaoliang Ye <supersingerman@126.com>; Mention <mention@noreply.github.com>
Subject: Re: [Microsoft/tslint-microsoft-contrib] Add react-a11y-proptypes rule. (#241)

 

In test-data/ReactA11yProptypes/FailingTestInputs/notAllowUndefined.tsx <https://github.com/Microsoft/tslint-microsoft-contrib/pull/241#discussion_r78519606> :

> @@ -0,0 +1,4 @@
> +import React = require('react');

The expression { } is undefined for our function isUndefined(), but there have an error "error TS17000: JSX attributes must only be assigned a non-empty 'expression'".
Is is tslint error or other error ? I don't know. @ipip2005 <https://github.com/ipip2005>  @HamletDRC <https://github.com/HamletDRC>  

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub <https://github.com/Microsoft/tslint-microsoft-contrib/pull/241/files/218c0185370898b09b2a0370a2940269bb2a459e#r78519606> , or mute the thread <https://github.com/notifications/unsubscribe-auth/AHWr9x6OsIUHEclq2gAh6arm4sPrqocvks5qpmN6gaJpZM4J6hy3> .  <https://github.com/notifications/beacon/AHWr91ZwF982eUHTQvUIEyjnSTNBrlZRks5qpmN6gaJpZM4J6hy3.gif>